test/e2e/view-court-reports-contracted-hours.js

```diff
@@ -18,7 +18,6 @@ describe('View contracted hours for court reporter', async function () {
   })
 
   it('should navigate to the court reporter contracted-hours page', async function () {
```

Comment:
Is there much point in keeping this test? You can't get to this page any more

Reply:
I'm going to remove it in a separate PR